### PR TITLE
Update cl_emv_meta.lua

### PR DESCRIPTION
**Pull request by SentryGunMan**
added attachment parenting and bone hiding to props and cleaned up some stupid ass code

### CODE REVIEW

File: lua/autorun/photon/cl_emv_meta.lua [hunk 1]
```diff
@@ -707,10 +717,14 @@ function EMVU:MakeEMV( emv, name )
...
 					break
 				end
-				prop:SetParent( self )
-				prop:SetPos( self:LocalToWorld( emvProps[index].Pos ) )
-				prop:SetAngles( self:LocalToWorldAngles( emvProps[index].Ang ) )
-				prop:DrawShadow( false )
+				/*
+				potentially useless? idk im shit at coding but it gets called every frame
+				and really fucks with bone parenting - sgm
+				*/
```
Comment (JoshPiper) [inline lua/autorun/photon/cl_emv_meta.lua]: Please use standard lua comments.

File: lua/autorun/photon/cl_emv_meta.lua [hunk 2]
```diff
@@ -707,10 +717,14 @@ function EMVU:MakeEMV( emv, name )
...
+				--prop:SetParent( self )
+				--prop:SetPos( self:LocalToWorld( emvProps[index].Pos ) )
+				--prop:SetAngles( self:LocalToWorldAngles( emvProps[index].Ang ) )
+				--prop:DrawShadow( false )
```
Comment (JoshPiper) [inline lua/autorun/photon/cl_emv_meta.lua]: Could be to handle the desync over time in some vehicles? Not 100% sure though.

**Review by JoshPiper — CHANGES_REQUESTED** [review]

**Review by JoshPiper — APPROVED** [review]

**Review by JoshPiper — APPROVED** [review]
Approved for theory, not tested ingame.

**Review by SuperMighty1 — APPROVED** [review]
gamer

**Pull request merged**